src/addon.cpp

```diff
@@ -66,35 +66,39 @@ class SHA3Hash: public ObjectWrap {
 
 	static
 	NAN_MODULE_INIT(Init) {
-		Local<FunctionTemplate> t = Nan::New<FunctionTemplate>(New);
-		t->SetClassName(Nan::New<String>("SHA3Hash").ToLocalChecked());
-		t->InstanceTemplate()->SetInternalFieldCount(1);
+		Local<Context> context = target->CreationContext();
 
-		Nan::SetPrototypeMethod(t, "update", Update);
-		Nan::SetPrototypeMethod(t, "digest", Digest);
+		Local<String> className = Nan::New<String>("SHA3Hash").ToLocalChecked();
 
-		constructor.Reset(t->GetFunction());
-		target->Set(Nan::New<String>("SHA3Hash").ToLocalChecked(), t->GetFunction());
+		Local<FunctionTemplate> functionTemplate = Nan::New<FunctionTemplate>(New);
+
+		functionTemplate->SetClassName(className);
+		functionTemplate->InstanceTemplate()->SetInternalFieldCount(1);
+
+		Nan::SetPrototypeMethod(functionTemplate, "update", Update);
+		Nan::SetPrototypeMethod(functionTemplate, "digest", Digest);
```

Comment:
:memo: **Implementation Note:** This is the same as the previous L73 and L74 -- `t` was renamed to `functionTemplate` for clarity.